aborted_transactions_test.go

```diff
@@ -0,0 +1,717 @@
+// Copyright 2021 Google Inc. All Rights Reserved.
```

Comment:
I think this should be: 

```
// Copyright 2021 Google LLC
```

Reply:
Same for all other files.

Reply:
I'll do that in a separate update to prevent too many merge conflicts. As you mentioned, this should be changed in all the files, including files that are not included in this change.